src/app/view/FlagshipUI/README.md

```diff
@@ -0,0 +1,172 @@
+# `FlagshipUI` API Documentation
+
+
+## Nomenclature
+
+- `Status bar`: The phone's top bar where notification, clock and network info are displayed
+- `Navigation bar`: The phone's top bar where navigation buttons are displayed
```

Comment:
It is bottom bar no?

Reply:
Good catch. Fixed in https://github.com/cozy/cozy-flagship-app/compare/551b317fb1533f7d257fed4e5e05861ddd23a911..66ebb20b8471d1c760fb597d32e41c85574a5514